gh/config.yml

```diff
@@ -14,3 +14,4 @@ aliases:
 http_unix_socket:
 # What web browser gh should use when opening URLs. If blank, will refer to environment.
 browser:
+version: "1"
```

Comment:
I guess it indicates the version of the API but I cannot find the description about that 🤔

Reply:
anyway I will merge into `origin/master` :p